### DISCUSSION TITLE
Change linespread?

### DISCUSSION THREAD

**Discussion by jolars** [jolars/moloch#7] [Ideas]
The theme currently sets `\linespread{1.15}`. I am not completely sure this is a good idea. It gives a more airy feel, but I already think the default it fine. It also seems to lead to some spacing problems here and there.

**Comment by samcarter** [comment]
Just my personal opinion: I would not change the default `\linespread{1}`. Presentations normally don't have long blocks of text, instead they often use bullet points, images etc. Such elements already have a larger spacing without changing the line spread. If a user would like more line spacing, they can adjust it for themselves.

**Comment by jolars** [comment]
Thanks for the input! Yes, I agree. I'll make the change.
